packages/blade/src/components/Table/types.ts

```diff
@@ -354,6 +354,11 @@ type TablePaginationCommonProps = {
    * @default false
    */
   showLabel?: boolean;
+  /**
+   * The background color of the component.
+   * @default transparent
+   */
+  backgroundColor?: string;
```

Comment:
Hi @Sumitmaithani , thanks for the PR.

So the issue is not that the backgroundColor prop is missing. Its expected that we set backgroundColor internally without taking it as a prop.

The issue is that the backgroundColor is not being set when the `paginationType="server"` prop is there.

You can replicate the issue by setting that prop on TablePagination component.

```jsx
<Table
          data={data}
          pagination={
            <TablePagination paginationType="server" showPageSizePicker showPageNumberSelector />
          }
        >
```

Reply:
Hi @saurabhdaware,

Thank you for helping me understand the issue. I am new to this codebase. From my understanding, I tried to reproduce the issue here: [https://stackblitz.com/edit/hhwvwi-wm3jcu?file=App.tsx](https://stackblitz.com/edit/hhwvwi-wm3jcu?file=App.tsx). It appears that the TablePagination component has a default transparent background color, regardless of whether we pass paginationType="server".

Could you clarify what it means when we set the background color internally without taking it as a prop? Does that mean we override its CSS? I apologize if I am misunderstanding or bothering you.

Reply:
So we by default want to set the backgroundColor. We dont want to accept that as prop from consumer.

You can check how its being set in TableHeader. I think right now we're setting the background color as transparent for footer but it should be same as header so I think we just need to change the background color

Reply:
Yes, I have fixed it as @chaitanyadeorukhkar mentioned. We are now taking the background color from `useTableContext`.